### ISSUE TITLE
Unary plus sign not implemented?

### ISSUE THREAD

**Issue by aroberge** [brython-dev/brython#316]
The Unary plus sign appears not to be implemented (i.e. not parsed correctly):

```
class Test():
    def __pos__(self):
        print("plus")   # not working
    def __neg__(self):
        print("minus")
    def __invert__(self):
        print("invert")
a = Test()
+a
-a
~a
```

**Comment by PierreQuentel** [comment]
Merci André !

Shame no me, I didn't even know that `+a` was translated to `a.__pos__()`. 

I added this method to the built-in types that support it (int, float, complex - I realise I forgot booleans, maybe others...), and modified `getattr` to raise a `TypeError` (not an `AttributeError` !) for the types that don't.

**Comment by aroberge** [comment]
De rien!      

I was just doing experiments with magic methods for my "Karel the robot clone"  when I stumbled on this bug.

**Comment by PierreQuentel** [comment]
Implemented for booleans in commit 08eff0c9008698c3d3fb5cca63a139f581f4faec

**Comment by jrazik** [comment]
Hi,

It's seems that the problem appear again elsewhere.
In the console (and in my code), I wrote a dictionary for moving direction in this way, and here's what I got:

>>> X = {'g': -1, 'h': 0, 'd': +1, 'b': 0}
>>> X
{'g': -1, 'h': 0, 'd': -1, 'b': 0}

If I take off the '+', it works just fine.
